ml-agents/mlagents/trainers/models.py

```diff
@@ -30,8 +30,6 @@ class LearningRateSchedule(Enum):
 
 
 class LearningModel:
```

Comment:
LearningModelUtility ?

Reply:
I think if it's a concrete class (not just static methods) it shouldn't be a `*Utility`.

Reply:
Would it be better to just have it as methods in the models.py class and not as a class? I left it as a class to keep the methods organized. 

There aren't any non-static methods in this class - so Utility (or ModelUtilities) seems right.

Reply:
What was decided here ?